tests/test_buffer.py

```diff
@@ -86,6 +86,107 @@ def test_ConcreteBuffer_from_ndarray(self):
         self.assertTrue((ndarr == 0).all())
 
 
+class ConcreteBufferAlignmentTC(unittest.TestCase):
```

Comment:
Test cases are similar to expaner and collector and they have been verified the correctness.